### PR TITLE
IntegrationTest - safeGet

### PR DESCRIPTION
**Pull request by xlecours**
Integration test safe get

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,14 +28,14 @@ install:
...
-    - wget "http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar"
+    - wget "http://selenium-release.storage.googleapis.com/2.52/selenium-server-standalone-2.52.0.jar"
 
     - PHP_INI_SCAN_DIR=":test/" php -S localhost:8000 -t htdocs htdocs/router.php 2>1 > /dev/null & 
 
     # Start Selenium and redirect Selenium WebDriver
     # output to /dev/null so that it doesn't flood the
     # screen in the middle of our other tests
-    - DISPLAY=:99.0 java -jar selenium-server-standalone-2.45.0.jar  2>1 > /dev/null &
+    - DISPLAY=:99.0 java -jar selenium-server-standalone-2.52.0.jar  2>1 > /dev/null &
```
Comment (tedstrauss) [inline .travis.yml]: how do we know that travis has all the versions of selenium?

Comment (xlecours) [inline .travis.yml]: https://travis-ci.org/aces/Loris/builds/109452201 passed with selenium 2.52

Comment (driusan) [inline .travis.yml]: Travis doesn't have any version of Selenium, it's downloaded on another line with wget.. both lines need to be updated.

Comment (tedstrauss) [inline .travis.yml]: ok @driusan  . we're just trying to figure out why this PR isn't launching travis.

Comment (xlecours) [inline .travis.yml]: Got it. As  @driusan  said, it might just be related to some delay on Travis server. I re-sent the pull request and it is now queued

Comment (tedstrauss) [inline .travis.yml]: ah yes, a little white space always does the trick

File: modules/statistics/test/Statistics_Test.php [hunk 2]
```diff
@@ -24,13 +24,13 @@ public function testTabsFrameworkLoads()
             $link = $this->webDriver->findElement(WebDriverBy::PartialLinkText("Demographic Statistics"));
             $this->assertContains("Demographic", $link->getText());
         } catch(NoSuchElementException $e) {
-            print $this->webDriver->getPageSource();
+            print $this->getPageSource();
```
Comment (driusan) [inline modules/statistics/test/Statistics_Test.php]: Why does this change from webDriver to this?

**Pull request merged**